packages/editor/src/plugins/text/static-components/static-math.tsx

```diff
@@ -30,52 +46,35 @@ export function StaticMath({ src, inline }: StaticMathProps) {
         nowrap && 'whitespace-nowrap'
       )}
     >
-      {renderFormula(
-        addDisplayStyle ? '\\displaystyle ' + cleanedSrc : cleanedSrc
-      )}
+      {renderFormula(cleanedSrc)}
     </div>
   )
 
   function renderFormula(formula: string) {
-    // block formular use displaystyle
-    const html = formula
-      ? KaTeX.renderToString(formula, {
-          displayMode: false,
-          throwOnError: false,
-          strict: false,
-          macros: {
-            '\\Q': '\\mathbb{Q}',
-            '\\C': '\\mathbb{C}',
-            '\\and': '\\wedge',
-            '\\euro': '€',
-            '\\or': '\\vee',
-            '\\arccot': '\\operatorname{arccot}',
-            '\\sgn': '\\operatorname{sgn}',
-            '\\m': '\\text{ m}',
-            '\\cm': '\\text{ cm}',
-            '\\mm': '\\text{ mm}',
-            '\\km': '\\text{ km}',
-            '\\dm': '\\text{ dm}',
-            '\\l': '\\text{ l}',
-            '\\dl': '\\text{ dl}',
-            '\\cl': '\\text{ cl}',
-            '\\ml': '\\text{ ml}',
-            '\\s': '\\text{ s}',
-            '\\h': '\\text{ h}',
-            '\\D': '\\mathbb{D}',
-            '\\W': '\\mathbb{W}',
-            '\\L': '\\mathbb{L}',
-          },
-        })
-      : ''
+    if (!formula.length) return <span />
+
+    try {
+      const mathML = temml.renderToString(formula, {
+        displayMode,
+        macros,
+        throwOnError: false,
+        strict: false,
+        trust: false,
+      })
 
-    // Even though we can trust the html created by Katex we sanitize the html as a second guard against XSS.
-    const sanitizedHtml = DOMPurify.sanitize(html)
-    return (
-      <span
-        className="inline-block py-1 [page-break-inside:avoid]"
-        dangerouslySetInnerHTML={{ __html: sanitizedHtml }}
-      />
-    )
+      return (
+        <span
+          className={cn(
+            'inline-block pb-1 [page-break-inside:avoid]',
+            inline ? 'text-[1.1rem]' : 'text-[1.33rem]'
+          )}
+          dangerouslySetInnerHTML={{ __html: mathML }}
+        />
+      )
+    } catch {
+      // eslint-disable-next-line no-console
+      console.error('formula could not be rendered')
+      return <span />
```

Comment:
Maybe send error to Sentry as well so we as devs get notified about these errors directly?

Reply:
Good idea, but so far we have no Sentry support in the editor at all and I don't really feel like adding more serlo-only stuff right now :)

Reply:
I was thinking of only temporarily adding it (and only active on serlo.org) to check how much existing latex on serlo.org does not work.

Reply:
But, I see other ways to check that might be easier. For example, instead of rendering an empty <span>, rendering an error message and asking the user to let us know.

Reply:
Ah I see. This gives me an idea of bulk checking almost all formulas, then we don't need the users to unknowingly test it for us :)

Reply:
That would be optimal! :green_heart: 
In general, I think rendering an error message like "Sorry, your latex could not be rendered" instead of an empty `span` would be better for the user experience as well.

Reply:
Great news! That already sounds promising! :smiling_face_with_three_hearts:

Reply:
With the [new version of Temml](https://github.com/ronkok/Temml/issues/84#issuecomment-2773356219) all formulas render without throwing 🥳 
I also fixed some faulty formulas in the content so now everything (over 400k formulas 🤯) should render without errors.

Reply:
Awesome! :partying_face: